doc/tutorials/quickstart_in_rviz/launch/panda_hello_moveit.rviz

```diff
@@ -0,0 +1,504 @@
+Panels:
```

Comment:
Why do we need a new RViz config here?

Also, it's missing the `MarkerArray` widget subscribing to `/rviz_visual_tools`, if you wanna add that back in.

Reply:
I was very hesitant at first to add the second RViz config, especially since it required a rewrite of demo.launch.py (but I think it could be used as a good example of how to pass in strings as launch arguments). The reason I did so was the tutorials instruct the user to add an RvizVisualToolsGui panel and save the changes to overwrite the original file. I wanted to be able to run the launch commands in "Planning Around Objects" without having to include "add an RvizVisualToolsGui panel to RViz" step, so I made a second config file (which I just added the MarkerArray to).

An alternate approach I could take would be to modify the original config file when placing it in the container to include the RvizVisualToolsGui panel and MarkerArray

Reply:
I think that's fair. Let's keep it :)